python/pyspark/sql/tests.py

```diff
@@ -3806,6 +3806,7 @@ def test_timestamp_dst(self):
     not _have_pandas or not _have_pyarrow,
     _pandas_requirement_message or _pyarrow_requirement_message)
 class PandasUDFTests(ReusedSQLTestCase):
+
```

Comment:
seems unrelated ..

Reply:
It is, however makes it fit with the style of the rest of the file.